packages/esm-patient-registration-app/src/patient-registration/field/dob/dob.component.tsx

```diff
@@ -46,8 +47,8 @@ export const DobField: React.FC = () => {
   );
 
   const onDateChange = useCallback(
-    (birthdate: Date) => {
-      setFieldValue('birthdate', birthdate);
+    (birthdate: CalendarDate) => {
```

Comment:
Shouldn't this be of type `DateValue`? I thought that's what the onChange callback expects.

Reply:
I think that was before the framework was updated in the module.

Reply:
Yeah, for the DatePicker, at any rate, this will always be a `CalendarDate`. Would it be helpful to have a wrapper here that instead returned a normal JS date?

Reply:
I'm indifferent about that, as it's just one (albeit non-obvious) line to convert `CalendarDate` to JS date. I'd be happy to see more utilities in core to help handle datetime in general though. (so things like combining the input values of `DatePicker` and `TimePicker` into a single Date is standardized)